index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
```

Comment:
I think your heading class can be named something other than "header". The header is a semantic tag you can use to group your head section.
Here is a[ resource ](https://www.w3schools.com/html/html5_semantic_elements.asp)to help you learn more about HTML semantic tags

Thank you😎